sample/ChannelsSample/index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>WebSocket Tests</title>
+<body>
+<h2>WebSocket Echo Test</h2>
+<div id="output1"></div>
+<hr />
+<h2>WebSocket Sample Test</h2>
+<a href="/ping" target="_blank">Saturn Ping</a>
+<div id="output2"></div>
+
+<script>
+    // create a guid in javascript, taken from: https://stackoverflow.com/questions/105034/how-to-create-guid-uuid
+    function uuidv4() {
+        return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+            var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
+            return v.toString(16);
+        });
+    }
+    function writeToScreen(name,domElement){
+        return msg => {
+            console.log(name,msg);
+            let p = document.createElement('p');
+            p.style.wordWrap = 'break-word';
+            p.innerHTML = msg;
+            domElement.appendChild(p);
+        }
+    }
+
+    let doSend = (ws,writer,msg) => {
+        let display = typeof msg == "string" ? msg || typeof msg == "number" : JSON.stringify(msg);
+        writer('Sending: ' + display);
+        ws.send(msg);
```

Comment:
`ws.send` doesn't support automatic serialization - you need to do that before. In current version message sent to the server was a string `[object Object]` which obviously was not parsed successfully. 

```suggestion
        ws.send(display);
```